R/oauth-init.R

```diff
@@ -72,17 +75,17 @@ init_oauth2.0 <- function(endpoint, app, scope = NULL,
                           user_params = NULL,
                           type = NULL,
                           use_oob = getOption("httr_oob_default"),
+                          oob_value = NULL,
                           is_interactive = interactive(),
                           use_basic_auth = FALSE,
                           config_init = list(),
                           client_credentials = FALSE
                          ) {
 
   scope <- check_scope(scope)
-  use_oob <- check_oob(use_oob)
-
+  use_oob <- check_oob(use_oob, oob_value)
   if (use_oob) {
-    redirect_uri <- "urn:ietf:wg:oauth:2.0:oob"
+    redirect_uri <- ifelse(!is.null(oob_value), oob_value, "urn:ietf:wg:oauth:2.0:oob")
```

Comment:
```suggestion
    redirect_uri <- if (!is.null(oob_value)) oob_value else "urn:ietf:wg:oauth:2.0:oob"
```

`ifelse()` is specifically for creating vectors; you can just use an inline `if` here